doc/api/dgram.md

```diff
@@ -174,7 +174,8 @@ This method throws `EBADF` if called on an unbound socket.
 <!-- YAML
 added: v0.1.99
 changes:
-  - version: v0.10
+  - version: v0.9.1
```

Comment:
Is the version supposed to change here?

Reply:
`v0.10` is not a valid `semver` change, so I had to change it. According to GitHub UI, 332fea5ac1816e498030109c4211bca24a7fa667 was part of `v0.9.1` so I put that instead. I can make a separate PR for this if you think that's more appropriate.

Reply:
I think it's fine, maybe just related to the old uneven releases being "unstable", so it wasn't officially documented till the next stable (even)